.github/workflows/issue-auto-pr.yml

```
@@ -0,0 +1,97 @@
+          SANITIZED_TITLE=$(echo '${{ github.event.issue.title }}' | tr '[:upper:]' '[:lower:]' | sed 's/[^a-z0-9]/-/g' | sed 's/--*/-/g' | sed 's/^-\|-$//g' | cut -c1-50)
+          BRANCH_NAME="codex/issue-${{ github.event.issue.number }}-${SANITIZED_TITLE}"
+          
+          git checkout -b "$BRANCH_NAME"
+          
+          git add .
+          
+          if git diff --staged --quiet; then
+            echo "No changes were made by Codex. Exiting without creating PR."
+            exit 0
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)